meta-facebook/yv4-sd/src/platform/plat_pldm_monitor.c

```diff
@@ -90,3 +109,94 @@ uint8_t plat_pldm_get_state_effecter_state_handler(const uint8_t *buf, uint16_t
 
 	return PLDM_SUCCESS;
 }
+
+uint8_t plat_pldm_host_button_sequence(const uint8_t *power_sequence, uint16_t pressing_interval)
+{
+	uint8_t sts_cnt = 3;
```

Comment:
Lets define a macro and use that here and in power sequence and reset sequence definition rather than using arbitrary numbers at multiple places.

Reply:
Revised, thank you for the suggestion.